src/main/java/org/fundacionjala/at15/pokemon/Pokemon.java

```diff
@@ -2,13 +2,22 @@
 
 import static org.fundacionjala.at15.pokemon.io.EntityType.*;
 import java.util.ArrayList;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
 import org.fundacionjala.at15.pokemon.ID.Identifier;
 
+@XmlRootElement
 public class Pokemon extends Entity {
     private HitPoints hitPoints;
     private ArrayList<Move> movementList = new ArrayList<Move>();
     private String pokemonName;
 
+    public Pokemon() {
```

Comment:
Now are going to be 2 constructors?

Reply:
Yes, This library needs no arguments default constructor tu run